### PR TITLE
Fix Password validation on the Login/Signup Page [Fixes #720] 

### LINKED ISSUE #720: Unmatched password filed validation between Talawal mobile app &  Talwal admin
when i tried to register a new account `on the admin `portal i was able to register an account with a password which has the following validation **(at least 8 characters long ),** but when i tried to use the same password to login `using the mobile app` it gave me the error **(password must have at least 8 char, one upper Case , one number ,one special characters)**

**To Reproduce**

1. first choice is to edit the validation on the `mobile app` to (**at least 8 characters )**
2.second choice is to edit the validation on the `admin portal` to (**password must have at least 8 char, one upper Case , one number ,one special characters)**
 i am ready to be assigned for this issue

**Expected behavior**
matched password validation between mobile app & admin portal

**Actual behavior**
unmatched password validation between mobile app & admin portal

**Screenshots**
WhatsApp Image 2023-03-15 at 8 53 01 PM (1)
WhatsApp Image 2023-03-15 at 8 53 01 PM (2)
WhatsApp Image 2023-03-15 at 8 53 01 PM
WhatsApp Image 2023-03-15 at 8 53 02 PM

### PR DESCRIPTION
**Pull request by BhuvanAde**
**What kind of change does this PR introduce?** 
It is a bug fix; the password validation during the signup only constrain the password on being non-empty but now it will ensure that it is at least 8 characters long and has at least one capital letter, one number and one special character.

**Issue Number:**
#720 

Fixes #720 

**Did you add tests for your changes?**
Yes

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**
No

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-admin/blob/master/CONTRIBUTING.md)?**
Yes

### CODE REVIEW

**Comment by BhuvanAde** [comment]
This PR currently solves the issue, but it may be extended to have other language supports as well which needs discussion. The critical thing right now was syncing the mobile password rules with the admin panel, thus I have raised this PR.

**Comment by BhuvanAde** [comment]
@noman2002 I added some validations on the password field while registering on the admin portal.

**Review by ttaylor92 — CHANGES_REQUESTED** [review]
Please ensure to add the .yarn folder to the gitignore file so as to avoid pushing npm packages to repo. Update and resubmit for review

**Comment by BhuvanAde** [comment]
@ttaylor92 I Fixed that! please check

File: src/screens/LoginPage/LoginPage.tsx [hunk 1]
```diff
@@ -134,8 +134,12 @@ function LoginPage(): JSX.Element {
       signfirstName.length > 1 &&
       signlastName.length > 1 &&
       signEmail.length >= 8 &&
-      signPassword.length > 1
+      signPassword.length >= 8 &&
+      /[A-Z]/.test(signPassword) &&
+      /[0-9]/.test(signPassword) &&
+      /[^A-Za-z0-9]/.test(signPassword)
     ) {
+      console.log('yes');
```
Comment (Rumone) [inline src/screens/LoginPage/LoginPage.tsx:142]: Is this still being used?

**Review by Rumone — CHANGES_REQUESTED** [review]

File: src/screens/LoginPage/LoginPage.tsx (hunk 1 above)
Comment (BhuvanAde) [reply src/screens/LoginPage/LoginPage.tsx]: Removed!

**Comment by anshgoyalevil** [comment]
@BhuvanAde 

Please pull the latest code, and re-run the tests.

**Comment by BhuvanAde** [comment]
@ttaylor92 @Rumone I have sent another PR with the required changes and I'll close this PR. I request you to please review #881

**Pull request closed without merging**